tests/examples/literate/expected

```diff
@@ -1 +1,2 @@
 1/1: Building Source (Source.tex)
+  {\end{Verbatim}}
```

Comment:
I think you were meant to update `idris2-expected.sty` rather than accept this strange output